admin-manual/installation-setup/customization/scaling-archivematica.rst

```diff
@@ -0,0 +1,396 @@
+
+Processing configuration options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`Processing configuration <dashboard-processing>` screen provides many 
+options for controlling what actions Archivematica performs. The following 
+settings can improve performance. 
+
+**Select file format identification command (Transfer):** Using Siegfried for 
+file identification has been shown to be faster than Fido in this 
+`benchmarking`_ study.
+
+**Select file format identification command (Ingest):** If you are using the 
+Archivematica backlog and have accumulated items in the backlog for a long 
+period of time, e.g. months or even years, then you might want Archivematica to
+refresh its file format identification at the Ingest stage of the workflow. If
+Ingest is happening shortly after Transfer, selecting `Use existing data` 
+should be adequate and will save processing time. 
+
+**Generate thumbnails:** If you are generating thumbnails, selecting `yes 
+without defaults` will only generate thumbnail images for formats that have a 
+specific thumbnail rule defined. The default thumbnail rule only produces a 
+grey icon and for many formats has little value, but in transfers with many 
+files, can take significant processing time. 
+
+**Select compression algorithm:** AIP compression causes an AIP to be put into 
+a container (e.g. a 7Zip container).  Using containers makes AIP storage and 
+transfer easier because the AIP is easier to move around as a single file. The 
+AIP file size also has the potential to be reduced, which saves storage space 
+and speeds up transfers to external AIP stores. The disadvantage is that 
+compression can take significant processing time and resources. AIP compression 
+introduces three extra steps in the workflow: compression to create the 
+container, then decompression to allow for a final checksum validation step. In 
+transfers with very large numbers of files (thousands) we have seen significant
+performance improvements by not compressing the AIP.  
+
+**Select compression level:** Selecting a higher compression level means that 
+the resulting AIP is smaller, but compression also takes longer. Lower 
+compression levels mean quicker compression, but a larger AIP.
+
+Preservation Action Rules
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some of the default preservation action rules can take considerable processing 
+time and resources. We have found the following rules useful to change in some 
+cases. 
+
+**Turn off default characterization rule:** FITS is used to characterise files 
+that don't have a recognised file format. Executing this rule takes processing 
+time and adds raw output to the METS file that can be low value for some  
+formats. For example, in scientific datasets with large numbers of generic text 
+files, or binary files created by instruments in scientific experiments, the 
+output can be verbose without being useful. 
+
+**Reduce number of image characterization rules:** Archivematica has rules 
```

Comment:
image and also audio-visual materials use all three.

Reply:
good catch, have added mention of audio-visual formats too.